sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/main/scala/com/azure/cosmos/spark/CosmosClientConfiguration.scala

```diff
@@ -31,7 +32,8 @@ private[spark] object CosmosClientConfiguration {
       cosmosAccountConfig.key,
       applicationName,
       cosmosAccountConfig.useGatewayMode,
-      useEventualConsistency)
+      useEventualConsistency,
+      cosmosAccountConfig.preferredRegionsList.getOrElse(Array[String]()))
```

Comment:
Can this default be at the AccountConfig level when it's parsed?

Reply:
valid point. thanks Matias.

I changed to code to not have default outside of CosmosConfig.